advanced_functionality/scikit_bring_your_own/scikit_bring_your_own.ipynb

```diff
@@ -493,6 +499,88 @@
    "source": [
     "sess.delete_endpoint(predictor.endpoint)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Batch Transform Job\n",
+    "One can use trained model to get inference on large data sets by using [Amazon SageMaker Batch Transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html). A Batch Transform Job takes your input data S3 location and publishes prediction result to specified S3 output folder. Similar to hosting, lets extract inferences for training data to demo Batch Transform mechanism."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a Transform Job\n",
+    "We'll create an `Transformer` that defines how to use the container to get inference results on a data set. This includes the configuration we need to invoke SageMaker Batch transform:\n",
```

Comment:
We'll create an `Transformer` that defines how to use the container to get inference results on a data set. This includes the configuration needed to invoke SageMaker batch transform:\n",